### PR TITLE
Add getting started docs for Jaeger and Zipkin receivers

### PR DESCRIPTION
**Pull request by mxiamxia**
### Description
Add getting started docs for Jaeger and Zipkin receivers

### Test
built and reviewed the pages in local as expected

### CODE REVIEW

File: src/docs/components/jaeger-zipkin-receiver.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+title: 'Getting Started with the Jaeger and Zipkin Receivers'
+description:
+    A receiver is how data gets into the OpenTelemetry Collector. Generally, a receiver accepts data in a specified format,
+     translates it into the internal format and passes it to processors and exporters defined in the applicable pipelines.
+path: '/docs/components/jaeger-zipkin-receiver'
+---
+
+import SectionSeparator from "components/MdxSectionSeparator/sectionSeparator.jsx"
+
+Jaeger receiver and Zipkin receiver are now included as a part of the AWS Distro for OpenTelemetry(ADOT).
```
Comment (anuraaga) [inline src/docs/components/jaeger-zipkin-receiver.mdx:11]:
```suggestion
Jaeger receiver and Zipkin receiver are included as a part of the AWS Distro for OpenTelemetry(ADOT).
```

Comment (anuraaga) [inline src/docs/components/jaeger-zipkin-receiver.mdx:11]:
```suggestion
Jaeger receiver and Zipkin receiver are now included as a part of the AWS Distro for OpenTelemetry (ADOT).
```

File: src/docs/components/jaeger-zipkin-receiver.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+A receiver is how data gets into the OpenTelemetry [Collector](https://github.com/open-telemetry/opentelemetry-collector).
```
Comment (anuraaga) [inline src/docs/components/jaeger-zipkin-receiver.mdx:12]:
```suggestion
A receiver is how data gets into the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector).
```

File: src/docs/components/jaeger-zipkin-receiver.mdx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+ Generally, a receiver accepts data in a specified format, translates it into the internal format and passes it to
```
Comment (anuraaga) [inline src/docs/components/jaeger-zipkin-receiver.mdx:13]:
```suggestion
 Generally, a receiver accepts data in a specified format, translates it into the collector's internal format and passes it to
```

File: src/docs/components/jaeger-zipkin-receiver.mdx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+ processors and exporters defined in the applicable pipelines.An exporter is a component in the
```
Comment (anuraaga) [inline src/docs/components/jaeger-zipkin-receiver.mdx:14]:
```suggestion
 processors and exporters defined in the applicable pipelines. An exporter is a component in the
```

File: src/docs/components/jaeger-zipkin-receiver.mdx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+ OpenTelemetry  configured to send data to different systems/back-ends.
```
Comment (anuraaga) [inline src/docs/components/jaeger-zipkin-receiver.mdx:15]:
```suggestion
 OpenTelemetry configured to send data to different systems/backends.
```

File: src/docs/components/jaeger-zipkin-receiver.mdx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+<SectionSeparator />
+
+## Jaeger Receiver
+
+Jaeger receiver receives trace data in [Jaeger](https://www.jaegertracing.io/) format, translates it to OpenTelemetry traces format and
+the collector pipeline can pass the date to different backends. You can find more information in OpenTelemetry Collector repository for
```
Comment (anuraaga) [inline src/docs/components/jaeger-zipkin-receiver.mdx:22]:
```suggestion
the collector pipeline can pass the data to different backends. You can find more information in OpenTelemetry Collector repository for
```

File: src/docs/components/jaeger-zipkin-receiver.mdx [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:14260
+```
+
+<SectionSeparator />
+
+## Zipkin Receiver
+
+Zipkin receiver receives spans from [Zipkin](https://zipkin.io/) (V1 and V2), translates it to OpenTelemetry traces format and
```
Comment (anuraaga) [inline src/docs/components/jaeger-zipkin-receiver.mdx:51]:
```suggestion
Zipkin receiver receives spans in [Zipkin](https://zipkin.io/) (V1 and V2) format, translates it to OpenTelemetry traces format and
```

**Comment by mxiamxia** [comment]
@anuraaga, Thx for the help on reviewing.

**Review by alolita — APPROVED** [review]
lgtm. Will merge at launch.

**Pull request merged**